### DISCUSSION TITLE
Playlist grouped by artists

### DISCUSSION THREAD

**Discussion by surfstyle** [audacious-media-player/audacious#1536] [General]
I'm trying to understand if it's possible to group the songs by artists.
I have a playlist/library where i see all my music library (some Giga), and it's difficult to scroll throw it.
Having a playlist grouped by artists it's something that can be interesting

**Comment by mschwendt** [comment]
You can left-click the playlist's column titles, e.g. click "Artist" to sort by artist, click "Title" to sort by title, and so on.

**Reply by surfstyle** [reply]
yep, sorting is ok. I'm trying to find a solution for grouping all the songs of a album into 1 row. So the playlist is not a list of all the files, but only of albums/artists. In this case if someone has a big library, it's more simple

**Reply by mschwendt** [reply]
Ah, okay. Some way of collapsing/uncollapsing albums would be more interesting IMHO.

Either way (regardless of whether it's for the same artist, album, whatever), if current playlist position arrives at such a collapsed entry, the design could get awkward.
